smassh/ui/widgets/typing/space.py

```diff
@@ -91,7 +91,7 @@ def wrapper(space: "Space") -> RenderableType:
 
 def blind_mode(func):
     def wrapper(space: "Space", *args, **kwargs) -> Style:
-        if config_parser.get("blind_mode"):
+        if config_parser.get("blind_mode") in (True, "on"):
```

Comment:
I think you can just do a `== "on"` :)

Reply:
https://github.com/kraanzu/smassh/blob/8a6e5105f5f78398909cae33f9ccfbf4762b4043/smassh/ui/widgets/settings/option.py#L115-L116

Reply:
> I think you can just do a `== "on"` :) 

Possibly, but its best to account for every allowed behavior, and since you seem to be allowing both bools and "on"/"off" i figured I'll just do both. Also sorry if my sentences are formed a bit rude, today is / was just not my day

Reply:
> Also sorry if my sentences are formed a bit rude, today is / was just not my day

Happens man! No worries

> I suppose thats why its iportant to have code consistency.

Hmm.. I think I'll make the default config more consistent using "on"/"off" for every setting that might be boolean
So yes, I think you can remove the `True` for now :)

Reply:
Honestly, I personally prefer to use a boolean when I have a variable with only two allowed states, as at least in type-strong languages (wich python doesnt belong to, I know) this prevents invalid states from occuring in the first place, but you do you